task/sast-go/0.1/README.md

```diff
@@ -0,0 +1,14 @@
+# sast-go task (Skipped)
+
+## Description:
+
+sast-go task makes use of the **GoSec** tool, which is a type of Static Application Security Testing (SAST) tool that is specifically designed to analyze Go programming language code for potential security vulnerabilities.
+
+This helps in identifying a wide range of security issues, including buffer overflows, SQL injections, cross-site scripting (XSS), and other types of code injection attacks. By analyzing the code during the development phase, the GoSec tool can help identify potential vulnerabilities early in the development process, when they are less costly to fix.
+
+This task is currently skipped.
+## Results:
+
+| name               | description  |
+|--------------------|--------------|
+| HACBS_TEST_OUTPUT  | Test output. |
```

Comment:
```suggestion
| HACBS_TEST_OUTPUT  | Tekton task test output. |
```
Refer to https://github.com/redhat-appstudio/build-definitions/pull/331/files which has passed the review from doc writer. So you need to update the description for `HACBS_TEST_OUTPUT` in these three tasks.

Reply:
Done, thanks!